examples/otlp/README.md

```diff
@@ -10,6 +10,11 @@ SDK](https://github.com/open-telemetry/opentelemetry-cpp). The application then
 calls a `foo_library` which has been instrumented using the [OpenTelemetry
 API](https://github.com/open-telemetry/opentelemetry-cpp/tree/main/api).
 
+To enable tls authentication for OTLP grpc exporter, SslCredentials can be used by
+specifying the path to cacert pem file via OtlpExporterOptions. The path to such a
```

Comment:
```suggestion
specifying the path to client certificate pem file via OtlpExporterOptions. The path to such a
```

Reply:
Correct me - but ideally this would be client certificate file - containing client cer, client key and CA certificate chain, In that case, we can just say `client certificate` instead of `cacert` ?